js/admin.js

```diff
@@ -0,0 +1,10 @@
+/*
+ * This file is part of Flarum.
+ *
+ * (c) Toby Zerner <toby.zerner@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
```

Comment:
This file is _not_ part of Flarum :P.

Reply:
Definitely not! lol

I was just copying the [existing forum.js](https://github.com/FriendsOfFlarum/nightmode/blob/master/js/forum.js)!